happy-travel/edo
# AvailableCurrencies field was added to api models and db context model

Issue: https://github.com/happy-travel/agent-app-project/issues/1509
Insomnia Edo -> Admin -> Agencies -> System-Settings
Tested through insomnia

Test cases:
 1) Check if settings can gets and updates
 2) Check if new field availableCurrencies doesn't contain unallowed currencies
